.changeset/shaggy-fireants-try.md

```diff
@@ -0,0 +1,11 @@
+---
+'braid-design-system': minor
+'@braid-design-system/docs-ui': minor
+'site': minor
```

Comment:
FYI , we dont version the site and dont think there is a reason to cut a release of the system either.

Maybe just a `patch` for the `docs-ui` package?

```suggestion
'@braid-design-system/docs-ui': patch
```